include/velocypack/Slice.h

```diff
@@ -1100,9 +1116,135 @@ class Slice {
       char const* value;
       char binary[sizeof(char const*)];
     };
-    memcpy(&binary[0], _start + 1, sizeof(char const*));
+    memcpy(&binary[0], start() + 1, sizeof(char const*));
     return value;
   }
+
+  constexpr uint8_t tagOffset(uint8_t const* _start) const noexcept {
```

Comment:
The function parameter names shouldn't start with an underscore, to avoid confusion with the instance variable of the same name.
```suggestion
  constexpr uint8_t tagOffset(uint8_t const* start) const noexcept {
```